guides/doc-Managing_Hosts/topics/ref_customizing-the-registration-templates.adoc

```diff
@@ -29,6 +29,7 @@ endif::[]
 
 * The `host_packages` parameter is for installing packages on the host.
 * The `remote_execution_ssh_keys`, `remote_execution_ssh_user`, `remote_execution_create_user` and `remote_execution_effective_user_method` parameters are used in the `remote_execution_ssh_keys`. For more details se the details of the snippet.
+* The `encrypt_grub` parameter is to enable setting of an encrypted bootloader password for the host, the default value is `false`. To actually set the password, use the `grub_pass` macro in your template.
```

Comment:
```suggestion
* The `encrypt_grub` parameter is to enable setting of an encrypted bootloader password for the host, the default value is `false`.
To set a password, use the `grub_pass` macro in your template.
```

Reply:
Opsie daisy, thanks for catching that! :)